### PR TITLE
AndroidX migration and update gradle to latest version

### PR DESCRIPTION
**Pull request by pandelisgreen13**
Android X migration

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,7 @@
 buildscript {
   ext.versions = [
-    'supportLibrary': '28.0.0-alpha3',
+    'annotation_version': '1.0.0',
```
Comment (mattprecious) [inline build.gradle]: Can you actually just inline these now? The version isn't being re-used anymore so there's no point in keeping this separate.

**Comment by mattprecious** [comment]
The license hash in `.travis.yml` needs to be updated, too. I think it's `24333f8a63b6825ea9c5514f83c2829b004d1fee` now?

File: build.gradle (hunk 1 above)
Comment (pandelisgreen13) [reply build.gradle]: I fix it.

**Comment by pandelisgreen13** [comment]
> The license hash in `.travis.yml` needs to be updated, too. I think it's `24333f8a63b6825ea9c5514f83c2829b004d1fee` now?

I updated the hash with 24333f8a63b6825ea9c5514f83c2829b004d1fee.

File: build.gradle (hunk 1 above)
Comment (mattprecious) [reply build.gradle]: Sorry, I meant can these be deleted and put directly into the dependencies in `swirl/build.gradle`. 

Ex:
```
implementation "androidx.annotation:annotation:1.0.0"
```

Comment (pandelisgreen13) [reply build.gradle]: Oh sorry, i didn't understand you, i have pushed the fix.

**Comment by mattprecious** [comment]
Thanks!

**Pull request merged**